README.md

```diff
@@ -2347,6 +2350,17 @@ form-flow:
   path: 'name-of-file.yaml'
 ```
 
+#### Design System
+
+We are moving towards using a [custom theme](https://codeforamerica.github.io/uswds/dist/) of
```

Comment:
I might reword this. If I'm correct, it's actually setting the UWDS resolver to be used over the default, meaning that enabling it would turn on the USWDS resolver, but those templates will now replace the same ones from the Honeycrisp resolver? Meaning that turning this on will switch your application to using USWDS styles and components instead of Honeycrisp styles and components.